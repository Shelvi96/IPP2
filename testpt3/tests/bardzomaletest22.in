NEW ID

2>4:71;:76;6
6>49654
;84:>93121
1:60;9>;725
166:0>772301
98;;96574>;18:
;7>419250
13>143:46
507>544:
2;>928332848
@52301;:74:697:
@:331;5470;283:
@2705;
@8901;::794947:446386959;706;95
@:69;68
@595255091;5:5276226018::64
@92827581:
@602594:3
@4513:7309149
@;54
@2085525
@7
@4994579
