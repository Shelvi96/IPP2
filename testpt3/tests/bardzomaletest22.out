0
0
0
797161022046015
0
80384595016
0
0
0
0
0
0
0
